File: LetterTellerP/TurfTab.py
class TurfTab:
    def __init__(self):
        self.tellers = [0] * 26
        self.totaal = 0

    def turf(self, ch):
        if ord('A') <= ch <= ord('Z'):
            self.tellers[ch - ord('A')]+=1
            self.totaal+=1
        if ord('a') <= ch <= ord('z'):
            self.tellers[ch - ord('a')]+=1
            self.totaal+=1

    def Turf(self, s):
        for c in s:
            self.turf(ord(c))

    def __str__(self):
        res = ""
        for t in range(0,26):
            res += chr(t+ord('A')) + ": " + str(self.tellers[t]) + " keer\n"
        res += "totaal: " + str(self.totaal) + "\n"
        res += "gemiddeld: " + str(self.Gemiddelde)
        return res

    def berekenGemiddelde(self):
        return self.totaal / len(self.tellers)

    Gemiddelde = property(fget=berekenGemiddelde)

    def getWaardes(self):
        return self.tellers.copy()

    Waardes = property(fget=getWaardes)

File: LetterTellerC/TurfTab.cs
public class TurfTab
{
    protected int[] tellers;
    protected int totaal;

    public TurfTab()
    {
        tellers = new int[26];
    }
    protected virtual void turf(char ch)
    {
        if (ch >= 'A' && ch <= 'Z')
        {   tellers[ch - 'A']++;
            totaal++;
        }
        if (ch >= 'a' && ch <= 'z')
        {   tellers[ch - 'a']++;
            totaal++;
        }
    }
    public void Turf(string s)
    {
        foreach(char c in s)
            turf(c);
    }
    public override string ToString()
    {
        string res = "";
        for (int t = 0; t < 26; t++)
            res += (char)(t + 'A') + ": " + tellers[t] + " keer\n";
        res += "totaal: " + totaal + "\n";
        res += "gem.: " + Gemiddelde;
        return res;
    }
    public int Totaal
    {   get
        {   return totaal;
        }
    }
    public float Gemiddelde
    {   get
        {   return (float)totaal / tellers.Length;
        }
    }
    public int[] Waardes
    {   get
        {   return (int[])tellers.Clone();
        }
    }
}